Sources/Maker.swift

```diff
@@ -19,6 +19,13 @@ enum HandlerPriority: Int {
     case low
 }
 
+public enum Sides {
+    case top
+    case bottom
+    case left
+    case right
+}
```

Comment:
I can't understand how `vertical` and `horizontal` cases should work with insets. @artemnovichkov, Could you clarify it?

Reply:
I mean with `vertical` case it will use both left and right insets

Reply:
Well, it's sounds interesting

Reply:
I like vertical and horizontal 👍🏻. But think `all` cases will useless because `maker` have `public func edges(insets: UIEdgeInsets) -> Maker` method which can pin all sides.